### PR TITLE
meta-pine64-luneos: Initial work on bringing PinePhonePro device up

### PR DESCRIPTION
**Pull request by Herrie82**
This should bring the most functionality:
-Screen
-Wifi
-Brightness slider
-Volume buttons (working), power button (broken)
-Modem (powers on, not tested anything else yet)
-Bluetooth (seems to power on, haven't tested it further)
-Brightness sensor
-Audio playback
-Rotation (mostly) and most other sensors

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>

### CODE REVIEW

File: conf/machine/pinephonepro.conf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#@DESCRIPTION: PinePhone Pro based on Rockchip RK3399S Processor.
+
+require conf/machine/include/rock-pi-4.inc
+
+PREFERRED_PROVIDER_virtual/kernel = "linux-pinephonepro"
+
+KERNEL_DEVICETREE = "rockchip/rk3399-pinephone-pro.dtb"
+UBOOT_MACHINE = "pinephone-pro-rk3399_defconfig"
+
+IMAGE_FSTYPES += "wic.gz wic.bmap tar.gz"
```
[the remark this answers is not in the document]
Comment (Herrie82) [reply conf/machine/pinephonepro.conf:12]: I don't know, copied from PP ;)

File: recipes-connectivity/eg25-manager/eg25-manager.bb [hunk 2]
```diff
@@ -27,7 +28,10 @@ do_install:append() {
     install -m 0644 ${WORKDIR}/eg25-manager.service ${D}${systemd_system_unitdir}/eg25-manager.service
 }
 
-FILES:${PN} = "${sysconfdir}/udev/rules.d/80-modem-eg25.rules \
+FILES:${PN} = "/usr/udev \
+               /usr/udev/rules.d \
+               /usr/udev/rules.d/80-modem-eg25.rules \
```
[the remark this answers is not in the document]
Comment (Herrie82) [reply recipes-connectivity/eg25-manager/eg25-manager.bb:33]: For some reason it didn't want to work without before, let me try again

File: conf/machine/pinephonepro.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tofee) [reply conf/machine/pinephonepro.conf:12]: ah, maybe I kept it for easier debugging of the resulting image content, when I began working with PP. But it's not so useful now.
